src/e2e/step-definitions/then/verify-element-value.ts

```diff
@@ -139,31 +139,7 @@ Then(/^the "([^"]*)" contains the value "([^"]*)"$/, async (elementName: string,
 /* verify the attribute value of an element within another element */
 Then(/^the "([^"]*)" element within the "(1st|2nd|3rd|[0-9]+th)" "([^"]*)" element contains the "([^"]*)" attribute "([^"]*)"$/, async (subelement: string, parentElementIndex: string, mainElementName: string, attributeType: string, attribute: string) => {
   const index = parseInt(parentElementIndex, 10) - 1;
-  let element = await elementHelper().getElementInElementByCss(mainElementName, subelement, 0, true, index);
+  let element = await elementHelper().getElementInElementByCss(mainElementName, subelement, 0,true,index);
   const elementAttribute = await htmlHelper().getAttribute(element, attributeType);
   expect(elementAttribute).to.include(attribute);
 });
-
-Then(/^the "(1st|2nd|3rd|[0-9]+th)" "([^"]*)" (?:option|element|input) contains the text "([^"]*)"$/, async (elementPosition: string, elementName: string, expectedElementText: string) => {
```

Comment:
Did you move these steps to another file? We should not delete them

Reply:
fixed in latest merge